nixos/modules/services/databases/mysql.nix

```diff
@@ -483,7 +483,7 @@ in
             ''
               ( echo "CREATE USER IF NOT EXISTS '${user.name}'@'localhost' IDENTIFIED WITH ${if isMariaDB then "unix_socket" else "auth_socket"};"
                 ${concatStringsSep "\n" (mapAttrsToList (database: permission: ''
-                  echo "GRANT ${permission} ON ${database} TO '${user.name}'@'localhost';"
+                  echo 'GRANT ${permission} ON ${database} TO `${user.name}`@`localhost`;'
```

Comment:
I checked (code-wise) that this shouldn't break anything but somebody else should probably also check that. Also I would run some other modules that use this before merging.